src/data/Firefox/devtools.yaml

```diff
@@ -0,0 +1,6 @@
+name: Developer Tools
```

Comment:
I'm still not sure why we need the extra folders. Can we put all of these inside the `data` folder?

Reply:
because I need to keep track of the group each project belongs to.. do u think its better if i add it as a field in the yaml file?

Reply:
That's the crux of the issue I am getting at: do we even *need* to track what group it belongs to? My instinct says no.

Reply:
@eliperelman  in the current bugsahoy, the project is listed based on its group (e.g Firefox with its User Interface, Developer Tools, Internal, Internal (iOS), etc), or do you suggest changing the name to be Firefox : User Interface, Firefox: Developer Tools ? so we can place all of them in the same directory